### PR TITLE
feat: replace tailwind with @nuxtjs/tailwindcss

### PR DESCRIPTION
**Pull request by sgraewe**
## This PR does the following things:

- Removes tailwind-template files
- Adds https://github.com/nuxt-community/nuxt-tailwindcss

### CODE REVIEW

File: saofile.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ module.exports = {
       }
     }]
 
-    if (this.answers.ui !== 'none') {
+    if (this.answers.ui !== 'none' || this.answers.ui !== 'tailwind') {
```
Comment (clarkdo) [inline saofile.js]: Saojs will skip the action if dir is not existed, so this check can be removed

**Comment by clarkdo** [comment]
OK. Then it can be merged after unit test fix

**Comment by Atinux** [comment]
Please fix the failing test :)

**Comment by sgraewe** [comment]
Alright, will need some help with the tests :)

**Comment by clarkdo** [comment]
@sgraewe Let me fix them

**Comment by clarkdo** [comment]
Thanks

**Pull request merged**